adapter/spvadapter/spvadapter.cpp

```diff
@@ -369,28 +369,27 @@ static void TransactionCallback(const char *txid, int status,
     tr->update(txid, status, msg);
 }
 
-const char *SpvDidAdapter_CreateIdTransaction(SpvDidAdapter *adapter,
+bool SpvDidAdapter_CreateIdTransaction(SpvDidAdapter *adapter,
         const char *payload, const char *memo, const char *password)
 {
     if (!adapter || !payload || !password)
-        return NULL;
+        return false;
 
     if (!memo)
         memo = "";
 
     TransactionResult tr;
 
-    SpvDidAdapter_CreateIdTransactionEx(adapter, payload, memo, 1,
+    return SpvDidAdapter_CreateIdTransactionEx(adapter, payload, memo, 1,
         TransactionCallback, &tr, password);
-
-    tr.wait();
+    /*tr.wait();
```

Comment:
IF we don't use /* ... */ anymore, then delete them. Otherwise if not sure about that, then can keep it for recovery ?

Reply:
Can delete line 381: TransactionResult tr;